modmail/extensions/plugin_manager.py

```diff
@@ -0,0 +1,104 @@
+
+    def __init__(self, bot: ModmailBot) -> None:
+        super().__init__(bot)
+        self.all_extensions = PLUGINS
+
+    def get_black_listed_extensions(self) -> list:
+        """
+        Returns a list of all unload blacklisted plugins.
+
+        This method exists to override the one in extensions manager,
+        due to the fact that blacklisting plugins is not supported.
+        """
+        return []
+
+    @commands.group("plugins", aliases=("plug", "plugs"), invoke_without_command=True)
+    async def plugins_group(self, ctx: Context) -> None:
+        """Install, uninstall, disable, update, and enable installed plugins."""
+        await ctx.send_help(ctx.command)
+
+    @plugins_group.command(name="load", aliases=("l",))
+    async def load_plugin(self, ctx: Context, *plugins: PluginConverter) -> None:
+        """
+        Load plugins given their fully qualified or unqualified names.
+
+        If '\*' or '\*\*' is given as the name, all unloaded plugins will be loaded.
+        """  # noqa: W605
+        await self.load_extensions.callback(self, ctx, *plugins)
+
+    @plugins_group.command(name="unload", aliases=("ul",))
+    async def unload_plugins(self, ctx: Context, *plugins: PluginConverter) -> None:
+        """
+        Unload currently loaded plugins given their fully qualified or unqualified names.
+
+        If '\*' or '\*\*' is given as the name, all loaded plugins will be unloaded.
+        """  # noqa: W605
+        await self.unload_extensions.callback(self, ctx, *plugins)
+
+    @plugins_group.command(name="reload", aliases=("r",))
+    async def reload_plugins(self, ctx: Context, *plugins: PluginConverter) -> None:
+        """
+        Reload extensions given their fully qualified or unqualified names.
+
+        If an extension fails to be reloaded, it will be rolled-back to the prior working state.
+
+        If '\*' is given as the name, all currently loaded extensions will be reloaded.
+        If '\*\*' is given as the name, all extensions, including unloaded ones, will be reloaded.
+        """  # noqa: W605
+        await self.reload_extensions.callback(self, ctx, *plugins)
+
+    @plugins_group.command(name="list", aliases=("all", "ls"))
+    async def list_plugins(self, ctx: Context) -> None:
+        """
+        Get a list of all extensions, including their loaded status.
+
+        Grey indicates that the extension is unloaded.
+        Green indicates that the extension is currently loaded.
+        """
+        await self.list_extensions.callback(self, ctx)
+
+    # TODO: Implement install/enable/disable/etc
+
+    # This cannot be static (must have a __func__ attribute).
+    async def cog_check(self, ctx: Context) -> bool:
+        """Only allow server admins and bot owners to invoke the commands in this cog."""
+        return ctx.author.guild_permissions.administrator or await self.bot.is_owner(ctx.author)
+
+
+# Delete the commands from ExtensionManager that PluginManager has inherited
```

Comment:
Add a `HACK:` mark, in case we figure out a better way to this inheritance in future.